benchmarks/sa.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3                                                         
+
+import time, argparse
+import numpy as np
+import arkouda as ak
+
+TYPES = ('int64', 'float64', 'bool', 'str')
+
+def time_ak_sa( vsize,strlen, trials, dtype):
```

Comment:
Recommend adding type checking here if possible for params and return type

Reply:
type checking has been added.